src/modules/fundingManager/bondingCurve/abstracts/BondingCurveBase_v1.sol

```diff
@@ -342,8 +345,23 @@ abstract contract BondingCurveBase_v1 is IBondingCurveBase_v1, Module_v1 {
         if ((_protocolFee + _workflowFee) > BPS) {
             revert Module__BondingCurveBase__FeeAmountToHigh();
         }
-        protocolFeeAmount = _totalAmount * _protocolFee / BPS;
-        workflowFeeAmount = _totalAmount * _workflowFee / BPS;
+        // Calculate protocol fee amount if applicable
+        if (_protocolFee > 0) {
+            protocolFeeAmount = _totalAmount * _protocolFee / BPS;
```

Comment:
Couldnt you just set the protocolFeeAmount == 0 check here?

Reply:
Assuming you meant moving the check into the if statement codeblock

Reply:
yes